troubleshoot/monitoring/node-moves-outages.md

```diff
@@ -9,33 +9,93 @@ products:
   - id: cloud-hosted
 ---
 
-# Troubleshoot node moves and outages [ec-deployment-node-move]
+# Understanding node moves and system maintenance [ec-deployment-node-move]
 
-To ensure that your nodes are located on healthy hosts, we vacate nodes to perform routine system maintenance or to remove a host with hardware issues from service.
+To ensure that your deployment nodes are located on healthy hosts, we vacate nodes to perform routine system maintenance or to remove a host with hardware issues from service.
 
 All major scheduled maintenance and incidents can be found on the Elastic [status page](https://status.elastic.co/). You can subscribe to that page to be notified about updates.
 
 If events on your deployment don’t correlate to any items listed on the status page, the events are due to minor routine maintenance performed on only a subset of {{ech}} deployments.
 
-**What is the impact?**
+This document explains the "`Move nodes off of allocator...`" message that appears on the [activity page](../../deploy-manage/deploy/elastic-cloud/keep-track-of-deployment-activity.md) in {{ech}} deployments, helping you understand its meaning, implications, and what to expect.
 
-During the routine system maintenance, having replicas and multiple availability zones ensures minimal interruption to your service. When nodes are vacated, as long as you have high availability, all search and indexing requests are expected to work within the reduced capacity until the node is back to normal.
+![Move nodes off allocator](images/move_nodes_ech_allocator.jpeg)
 
-**How can I be notified when a node is changed?**
+::::{note}
+You can [configure email notifications](#email) to be alerted when this situation occurs.
+::::
 
-To receive an email when nodes are added or removed from your deployment:
+## Possible causes and impact [ec-node-host-outages]
 
-1. Follow the first five steps in [Getting notified about deployment health issues](../../deploy-manage/monitor/monitoring-data/configure-stack-monitoring-alerts.md).
-2. At Step 6, to choose the alert type for when a node is changed, select **CLUSTER HEALTH** → **Nodes changed** → **Edit alert**.
+Potential causes of system maintenance include, but not limited to, situations like:
 
-::::{note}
-If you have only one master node in your cluster, during the master node vacate no notification will be sent. Kibana needs to communicate with the master node in order to send a notification. One way to avoid this is by shipping your deployment metrics to a dedicated monitoring cluster, which you can configure in Step 2 of [Getting notified about deployment health issues](../../deploy-manage/monitor/monitoring-data/configure-stack-monitoring-alerts.md) when you enable logging and monitoring.
+* A host where the Cloud Service Provider (CSP), like AWS, GCP, or Azure, has notified us in advance about hardware issues.
```

Comment:
Do we get an advance notice about hw issues? Even if we do, I think the term "advance" might set wrong expectations here, i.e. even with advance notice we won't be able to delay this operation.

Reply:
> about hardware issues

➕ this wording needs to be careful if included. Something like the cloud host may notify us of "pending hardware deprecation/maintenance that they'll be performing soon".

Reply:
Updated, let me know your thoughts @jakommo and @stefnestor. I've used Stef's suggestion.
Great feedback about the `advance` wording.

Reply:
Let's discuss this internally.
See elastic/support-tech-lead#1576 (comment) - [2] part